console/src/components/UIKit/theme/index.ts

```diff
@@ -287,3 +377,6 @@ export const theme = {
   alertBox,
   icon,
 };
+
+type _Theme = typeof theme;
+export interface Theme extends _Theme {}
```

Comment:
I didn't quite understand the use case of this.  We could have export all those types above individually as inline.

Or, I misunderstood you. Can you elaborate on this a bit? Thanks! @beerose

Reply:
Yes. So I think it's good to have type of the whole `theme` exported. As we may want to do for example:
```ts
const newTheme: Theme = {...}
```